src/popup/pages/AccountDetails.vue

```diff
@@ -167,16 +179,59 @@ export default defineComponent({
       }
     });
 
+    const showSearchBar = computed<boolean>(() => {
+      if (
+        searchTerm.value || (
+          clientHeight.value > initialClientHeight.value
+          && appInnerScrollTop.value >= 120
+        )
+      ) return true;
+
+      return false;
+    });
```

Comment:
Could be simplified to 
```suggestion
    const showSearchBar = computed<boolean>(() => (searchTerm.value || (
          clientHeight.value > initialClientHeight.value
          && appInnerScrollTop.value >= 120
        ))
    });
```